src/engine/sidechain/engineshoutcast.cpp

```diff
@@ -401,7 +418,7 @@ bool EngineShoutcast::processConnect() {
 
         m_iShoutFailures++;
         qDebug() << "Streaming server failed connect. Failures:" << shout_get_error(m_pShout);
-        sleep(1);
+        QThread::msleep(100);
```

Comment:
Actually these sleeps are just for if there is network/server problem. Wait a small amount time to make network/server settle.